OpenRA.Game/Selection.cs

```diff
@@ -42,8 +43,19 @@ public void Combine(World world, IEnumerable<Actor> newSelection, bool isCombine
 					actors.SymmetricExceptWith(adjNewSelection);
 				else
 				{
+					if (stickyselect == true)
+					{
+						if (newSelection.Count() == 0)
+						{
+							actors.UnionWith(adjNewSelection);
+						}
+						else
+						actors.Clear();
+						actors.UnionWith(newSelection);
+					}
+					else
 					actors.Clear();
-					actors.UnionWith(adjNewSelection);
+					actors.UnionWith(newSelection);
```

Comment:
This block needs one more level of indentation, and it has to be wrapped in braces, too.